tripleblindmarket/private-kit
# Sameer develop828

#### Description:

This PR is regarding to SAF-828

Added Notification ID to avoid push notification

#### Linked issues:

SAF-828

#### Screenshots:



#### How to test:

Steps:
1- Open the app
2- Go to the third tab
3- Disable location
4- Go to the first tab

Expected result:
The notification "Location Tracking Was Disabled" should be display once.
